src/utils/local.ts

```diff
@@ -104,15 +107,46 @@ export class LocalRelease extends EventEmitter implements Release {
     if (path.endsWith(configFile)) {
       const conf = this.getConfig()
       this.config = conf.config
-      this.files = conf.files || []
       this.app = conf.app || conf.app_id || ""
+      this.getFiles(conf.files)
     } else if (path.endsWith(secretsFile)) {
       this.secrets = this.getSecrets()
     }
     this.emit('update', this)
   }
+
+  private getFiles (files: string[] | undefined) {
+    if (!files) return
+
+    const conf = this.getConfig()
+    if (conf.files) conf.files.forEach((file) => {
+      if (!this.isIn(this.files, file)) this.files.push(file)
+    })
+
+    files.forEach((file) => {
+      glob(file, {}, (error, filesFound) => {
+        if (error) throw error
+
+        filesFound = filesFound.map((e) => this.removePrefix(e))
+        filesFound.forEach((file) => {
+          if (!this.isIn(this.files, file)) this.files.push(file)
+        })
```

Comment:
You can chain those 2 methods like `filesFound.map().forEach()`. Not a big deal, might be cleaner :) (spread over 2-3 lines)